config/routes.rb

```diff
@@ -1,56 +1,6 @@
 Rails.application.routes.draw do
-  # The priority is based upon order of creation: first created -> highest priority.
-  # See how all your routes lay out with "rake routes".
+  get 'welcome/index'
+  get 'welcome/about'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.